tests/unit/via/views/view_pdf_test.py

```diff
@@ -140,6 +145,34 @@ def test_it_can_stream_an_empty_iterator(self, http_service, call_view):
         assert isinstance(response, HTTPNoContent)
 
 
+@pytest.mark.usefixtures("secure_link_service", "pdf_url_builder_service")
+class TestJSTORPDF:
+    @pytest.mark.usefixtures("http_service")
+    def test_it(self, call_view, jstor_api, pyramid_request):
+        response = call_view(
+            "jstor://DOI", view=proxy_jstor_pdf, params={"jstor.ip": "1.1.1.1"}
+        )
+
+        assert response.status_code == 200
+        assert response.headers["Content-Disposition"] == "inline"
+        assert response.headers["Content-Type"] == "application/pdf"
+        assert (
+            response.headers["Cache-Control"]
+            == "public, max-age=43200, stale-while-revalidate=86400"
```

Comment:
I guess these are the default caching headers that we use for all PDFs? I do think some caching will be beneficial here, as PDFs can be large and it takes time for the JSTOR API to serve them. This caching will have some impact on eg. any analytics that JSTOR does, which we'll need to account for.

Reply:
Yes this are the defaults for every PDF.